### DISCUSSION TITLE
Normalisation methods (Global Vs RT Vs LFQ)

### DISCUSSION THREAD

**Discussion by witszymanski** [vdemichev/DiaNN#503] [General]
Good morning!
Firstly, thank you guys for the software! It's blazingly fast and easy to use. However, in order to convince my colleagues to switch our standard procedures to DiaNN, I wanted to clarify some things. I was analysing some data and was testing different settings, and I found out some interesting and puzzling things. Would be great if you could confirm and/or clarify my findings. 

### Experimental design
In this experiment, we have 4 different conditions (+ blanks) with 3 replicates each. 

I performed this DiaNN runs (all default settings with`unrelated runs` ticked):
1. RT norm + LFQ (standard)
2. Global + LFQ
3. RT norm
4. Global
5. LFQ
6. no norm
NOTE: I also run all these settings with some additional blank samples, just to see the behaviour of the very low intensity samples, that do not fall in general under the LFQ assumption. 

### Global / RT Vs LFQ
How do these normalisations work together/separate? You can use one, or another, or both. In the `report.tsv` we get separate columns for `normalised` and `MaxLFQ`.  But used normalisation seems to also affect the LFQ values. In this example below, if the MaxLFQ is ON, but Global/RT is off, we get this behaviour for blank samples.

On the graphs, we will have all protein group intensities across measured samples. 2 protein markers, interesting to us, that we can follow are marked with red and blue lines.

![image](https://user-images.githubusercontent.com/15866068/190970425-a433cf7b-f7c4-42ea-a999-a221e42c0a01.png)

But, is we turn on the Global/RT normalisation, the MaxLFQs are completely different. Mark, that all the blank sample MaxLFQ values are now pushed up. 

![image](https://user-images.githubusercontent.com/15866068/190970512-7e537bee-eb9a-4fa8-8da7-93ea588d2d12.png)

Can you elaborate about this situation? 

### Differences between LFQ with DiaNN and MaxQuant
Connected to the point above, I observed, that the LFQ values in MaxQuant are differently calculated. Here below, it is not the same dataset, but the basic idea is the same, that the samples on the far right are almost empty (it's a dilution curve). If we take a look at the not normalised intensities from MaxQuant DDA runs, we see the drop from left to right:
![image](https://user-images.githubusercontent.com/15866068/190994098-e8a45ec4-cb17-4655-935b-d56be769f491.png)

If we use the LFQ values, we see again the "push up" effect, which is not the case with DiaNN when we use MaxLFQ only:

![image](https://user-images.githubusercontent.com/15866068/190994219-1fa3b632-46aa-4a59-943d-bedb234ff239.png)

### Values in `report.pr_matrix.tsv`
Which values are used in the `report.pr_matrix.tsv` in the context of used normalisation and LFQ?  The blanks are pushed up if Global/RT normalisation was used.

![image](https://user-images.githubusercontent.com/15866068/190969478-159d285e-4dab-4977-8912-b25c862d05ca.png)

I do not really see any differences in the precursor matrix with the MaxLFQ switched ON (unless I made a mistake somewhere).
![image](https://user-images.githubusercontent.com/15866068/190969655-d07af56e-7a23-4ed5-9bf0-ea612da0bf23.png)

Blanks are in low intensity range if no Global/RT used. 
![image](https://user-images.githubusercontent.com/15866068/190969535-5c9922b1-07b9-4e0e-9156-03619e50b8b2.png)

### Empty values in `report.pg_matrix.tsv`
Whenever I switch off the LFQ (`--no-maxlfq `) my pg_matrices are always empty. Does this mean, that `report.pg_matrix.tsv` only always include vales based on MaxLFQs? 
![image](https://user-images.githubusercontent.com/15866068/190964014-b185618e-e438-419c-9a7c-cafd4bc798cc.png)

### Questions
It would be great, if someone can comment on my questions. I was looking for it on the github, but I have never found a full answers for my doubts. 

Have a good day

Witold Szymanski
Institute of Translational Proteomics
Biochemical/Pharmacological Center Department of Medicine
Philipps-Universität Marburg

**Comment by witszymanski** [comment]
Uploading the log.txt files, in case there was something strange with the settings I used. 
[globalRT-LFQ-report.log.txt](https://github.com/vdemichev/DiaNN/files/9598036/globalRT-LFQ-report.log.txt)
[LFQ-report.log.txt](https://github.com/vdemichev/DiaNN/files/9598039/LFQ-report.log.txt)
[Global-report.log.txt](https://github.com/vdemichev/DiaNN/files/9598041/Global-report.log.txt)

**Comment by t-jumel** [comment]
Hi, please excuse if I not answer all of your questions directly, 
if it is ok I would just drop a few comments here in case it helps.

MaxLFQ describes how precursor quant. was translated to protein group quant.,
therefore only protein groups and similar but not precursors carry MaxLFQ values.
It is much better than e.g. Top3 method and is default for the values displayed in the reduced output matrices.

I did a brief benchmark test involving a dilution to test different normalizations
in regards to deal with not perfectly equal sample loadings/ MS intensities.

I can recommend DIA-NN1.8 or higher with RT-dep. Norm. and MaxLFQ.
Global Norm. might have a small risk to overcorrect similar to MaxQuant from a few years ago.

Thats it already, the above recommendation should be good for most typical LFQ experiments.
Everyone else is naturally welcome to still comment as well.

Best, tobias

**Reply by witszymanski** [reply]
Hi Tobias! Thanks for that answer. Maybe someone else will comment on the rest.

Which values are displayed in the precursor matrix when LFQ is switched off (it's not empty like pg matrix)? Would be cool to have it somewhere in the github description. 

What is the relation between the Global/RT and LFQ normalisation? if they are all switched ONm does the global/rt work first, and the LFQ is additionally applied on top of it? Like a 2 stages of normalization? 

How does the Global/RT normalisation really work? Can I get my hands on the equation? 

Greets
Witek
